src/util.ts

```diff
@@ -134,12 +134,36 @@ export async function evaluate<T>(client: Client, fn: string, ...args: any[]): P
   const {Runtime} = client
   const jsonArgs = JSON.stringify(args)
   const argStr = jsonArgs.substr(1, jsonArgs.length - 2)
-  const expression = `(${fn})(${argStr})`
+
+  const expression = `
+    (() => {
+      const expressionResult = (${fn})(${argStr});
+      if (expressionResult && expressionResult.then) {
+        expressionResult.catch((error) => { throw new Error(error); });
```

Comment:
I'm trying to remember why this catch is needed... I'm pretty sure it has to do with errors in Chrome needing to be thrown explicitly to make it through CDP